### DISCUSSION TITLE
Is poetry necessary for this?

### DISCUSSION THREAD

**Discussion by PythonAI1234** [ig-python/ig-markets-api-python-library#325] [Q&A] [answered]
Because I always encounter the  can't import config from config file error, I am trying to reinstall everything. 
But now for poetry add trading-ig it says
invalid version 'poetry add trading-ig' on package trading-ig.
I just can't seem to get this library to work.
pleaase help.

**Comment by breitdom** [comment, marked as answer]
I can only recommend that you deal with these specific problems yourself. You can see from the code whether and where a library is needed. Try to install the library properly, Google can help.

There are even bigger challenges waiting for you further down the line.


Best regards
